source/deploy/mattermost-changelog.md

```diff
@@ -15,6 +15,82 @@ From Mattermost v9.2, this changelog summarizes updates for the latest cloud and
 :depth: 2
 ```
 
+## Release v9.4 - [Feature Release](https://docs.mattermost.com/upgrade/release-definitions.html#feature-release)
+
+**Release Date: January 16, 2024**
+
+### Important Upgrade Notes
+ - MySQL v5.7 is at end of life. We recommend all customers to upgrade to at least 8.x. For now, we are logging a warning. From Mattermost v9.5, which is the next Extended Support Release, we will stop supporting MySQL v5.7 altogether.
+
+```{Important}
+If you upgrade from a release earlier than v9.3, please read the other [Important Upgrade Notes](https://docs.mattermost.com/upgrade/important-upgrade-notes.html).
+```
+
+### Compatibility
+ - Updated the minimum required Edge version to v118+.
+
+### Improvements
+
+#### User Interface (UI)
+ - Updated the pre-packaged GitHub plugin version to [v2.1.7](https://github.com/mattermost/mattermost-plugin-github/releases/tag/v2.1.7).
+ - Pre-packaged Calls plugin version [v0.22.1](https://github.com/mattermost/mattermost-plugin-calls/releases/tag/v0.22.1).
```

Comment:
This will be v0.22.2 as of https://github.com/mattermost/mattermost/commit/baa88a2450d49dfea42990861283b5ed9d469edf